CMakeLists.txt

```diff
@@ -6,18 +6,12 @@ enable_testing()
 project(adobe_source_libraries CXX)
 
 set(CMAKE_CXX_EXTENSIONS OFF)
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_XCODE_ATTRIBUTE_CLANG_CXX_LANGUAGE_STANDARD "c++0x")
 set(CMAKE_XCODE_ATTRIBUTE_CLANG_CXX_LIBRARY "libc++")
 
 add_definitions ("-Wall")
-add_definitions ("-Werror")
-
-if (${CMAKE_CXX_COMPILER_ID} STREQUAL "Clang" AND
-    ${CMAKE_SYSTEM_NAME} MATCHES "Linux")
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -stdlib=libc++")
-    set(CMAKE_EXE_LINKER_FLAGS "-lc++abi")
-endif()
+#add_definitions ("-Werror")
```

Comment:
I should have commented that, but it's because of boost.filesystem calling a deprecated posix.
good time to move to std::filesystem ?